android/src/main/java/com/reactcommunity/rndatetimepicker/RNDatePickerDialogFragment.java

```diff
@@ -114,6 +115,7 @@ static DatePickerDialog createDialog(
     }
 
     final DatePicker datePicker = dialog.getDatePicker();
+	dialog.setOnShowListener(ensureButtonsVisible(activityContext, dialog));
```

Comment:
nit, but your IDE indentation seem to be tabs and not spaces like the rest of these files

Reply:
yeah, the project is a mess when it comes to this... I need to add editorconfig and unify this across the project. iOS project also has it messed up 🙈